packages/react-components/src/components/styles/ParticipantItem.styles.ts

```diff
@@ -14,17 +14,17 @@ export const participantItemContainerStyle = mergeStyles({
 });
 
 export const menuButtonContainerStyle = mergeStyles({
-  root: {
-    '&:hover': { background: 'none' },
-    '&:active': { background: 'none' }
-  }
+  paddingTop: '0.25rem'
 });
 
 export const iconContainerStyle = mergeStyles({
   display: 'flex',
   alignItems: 'center',
-  height: '100%',
-  overflow: 'hidden'
+  height: '100%'
 });
 
-export const iconStyles = mergeStyles({ display: 'flex', margin: 'auto' });
+export const iconStyles = mergeStyles({
+  display: 'flex',
+  lineHeight: 0,
```

Comment:
what does line height 0 do here? put a quick comment here to ensure this isn't removed in the future and the bug re-introduced
```suggestion
  lineHeight: 0, // ensure the icon ...
```

Reply:
with line-height at 0
![image](https://user-images.githubusercontent.com/82416644/133154862-981da815-b461-477b-aa87-454569cd2cdc.png)

without line-height at 0, the "More" icon is slightly above center line
![image](https://user-images.githubusercontent.com/82416644/133154736-c2bfdb71-b8b1-4fd4-a868-8504a5b861c8.png)